.github/workflows/weekly-spec-update.yaml

```diff
@@ -8,25 +8,59 @@ jobs:
   update-all-specs:
     runs-on: ubuntu-latest
     permissions:
-      contents: write
-      pull-requests: write
+      actions: write
```

Comment:
Drive-by fix because it was [failing for weeks](https://github.com/SAP/ai-sdk-java/actions/runs/17181678956)